ably/realtime/connection.py

```diff
@@ -194,6 +196,19 @@ def try_connect(self):
         self.connection_attempt_task = asyncio.create_task(self._connect())
         self.connection_attempt_task.add_done_callback(self.on_connection_attempt_done)
 
+    def __generate_fallback_hosts(self):
+        if len(self.options.get_fallback_realtime_hosts()) > 0:
+            for host in self.options.get_fallback_realtime_hosts():
+                yield host
+
+    def use_fallback_host(self):
```

Comment:
This method isn't public so should start with an underscore

Reply:
It's used in `websockettransport.py` for retry with unreachable host